api/openapi-spec/swagger.json

```diff
@@ -45699,7 +45699,44 @@
     }
    },
    "io.k8s.apimachinery.pkg.apis.meta.v1.Patch": {
-    "description": "Patch is provided to give a concrete name and type to the Kubernetes PATCH request body."
+    "description": "Patch is provided to give a concrete name and type to the Kubernetes PATCH request body.",
```

Comment:
Without looking more into this issue yet, you can't directly edit OpenAPI spec. Need to fix it in the source that spec is generated from.